io.podman_desktop.PodmanDesktop.yml

```
@@ -0,0 +1,95 @@
+app-id: io.podman_desktop.PodmanDesktop
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: run.sh
+separate-locales: false
+finish-args:
+  - "--socket=session-bus"
+  - "--socket=wayland"
+  - "--socket=x11"
```

this is not the fallback you mention.
```suggestion
  - "--socket=fallback-x11"
```